### ISSUE TITLE
Is it possible to connect to overmind in an existing tmux session

### ISSUE THREAD

**Issue by typhoon2099** [DarthSim/overmind#122]
I have a Procfile with three jobs, and I was hoping to be able to have a window in my tmux session displaying each of these jobs in a separate pane. I'm used `overmind connect` to rearrange those windows into panes in a single window, but it does not seem possible to move that window into a separate session.

Would I be better off using hivemind and writing a script to set up my session/window/panels myself?

TL;DR I think it would be cool to connect overmind as panes in the current window.

**Comment by DarthSim** [comment]
Hey @typhoon2099!
Overmind isn't designed to do so. First of all, managing processes in tmux is not an easy task, and I'm not sure how it will work in an existing session. Also, environment variables may be a problem. Currently, Overmind launches a new tmux server, so the environment would match in most cases. In case we connect to an existing session, tmux environment, and the environment Overmind was launched from most probably will differ.

**Comment by typhoon2099** [comment]
No worries, I wasn't sure if it was possible or not.
